### DISCUSSION TITLE
Please add the minValue props for line chart/area chart

### DISCUSSION THREAD

**Discussion by truongnv98** [Abhinandan-Kushwaha/react-native-gifted-charts#915] [Ideas]
Please add a min value option so that the chart can start drawing from the min value instead of 0
![Screenshot 2024-11-27 at 09 21 04](https://github.com/user-attachments/assets/2fe6f2a5-6439-4ac6-ad9a-7c30959d9652)

**Comment by Abhinandan-Kushwaha** [comment]
Yes @truongnv98 you can use the `yAxisOffset` prop for this.

**Reply by truongnv98** [reply]
Thank you! Let me try

**Reply by truongnv98** [reply]
I found my previous problems. I have set yAxisOffset > the min value of the chart so the chart doesn't display correctly

**Reply by truongnv98** [reply]
Thanks @Abhinandan-Kushwaha
